Cargo.toml

```diff
@@ -48,7 +48,7 @@ nightly = []
 bench = []
 
 [build-dependencies]
-built = "0.2.0"
```

Comment:
Since this is an API-compatible change, "^0.2" should be enough and pick-up this and any further updates which are compatible.

Reply:
I don't know enough about how the project likes to manage its `Cargo.toml` versioning so I'll defer this. Seems reasonable enough to me, though.

Reply:
I just bumped to 0.2.2, which updates some dependencies to more recent versions.

Reply:
https://github.com/jwilm/alacritty/pull/932/commits/a397443886d9f860faee9518af74758eb75086fb